src/components/DigitalCredentials/CredentialsLanding.vue

```diff
@@ -42,17 +42,20 @@
     <article id="digital-credentials-table">
       <v-row no-gutters>
         <v-col cols="12" lg="11">
-          <CredentialsTable />
+          <CredentialsTable
+            :issuedCredentials="issuedCredentials"
+          />
         </v-col>
       </v-row>
     </article>
   </section>
 </template>
 
 <script lang="ts">
-import { Component, Emit, Vue } from 'vue-property-decorator'
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
```

Comment:
Please `import Vue from 'vue'` instead (to avoid conflicts with my PR).

Reply:
Updated!

Reply:
Edit* for all added components